prov/efa/test/README.md

```diff
@@ -0,0 +1,62 @@
+# EFA unit tests
+
+## How to run
+
+To run efa unit tests, you will need to have cmocka installed.
+See: https://cmocka.org/
+
+You will need to configure libfabric with ```--enable-efa-unit-test```.
+
+An example build and run command would look like:
+
+```
+./autogen.sh; ./configure --enable-efa-unit-test; make check;
+```
+
+## How to write
+
+1. Create or find an existing c file for your class of unit tests.
+2. Add headers to efa_unit_tests.h for each new unit test.
+3. Create or find an existing cmocka test group in efa_unit_tests.c
+4. Add your new unit test to the cmocka test group.
+
+## Mocking
+
+To mock a function, you will need to add ```-Wl,--wrap=<function to mock>```
+to the Makefile.include as part of efatest_LIBS. Then, after declaring the function,
+you can replace it with __wrap_<function to mock>. If you need to use the original
+function, you can use __real_<function to mock> after declaring it.
+
+### Mocking static functions
+
+Static functions are not mockable due to the way they are compiled. Instead you
+will have to wrap a static function in a non static function. Currently we have
+no examples of this. If you would like to mock a static function, To make this
+simpler, you can add a macro "unit_static" that will change static functions to
+non static when --enable-efa-unit-test is enabled. You will also need to add
```

Comment:
We agreed on a different outcome from last week's meeting. Do we still want to mock static functions using 'unit_static' macro?

Reply:
I didn't update the PR since that meeting

Reply:
Removed.